apps/super-rentals/public/assets/bookmarklet.js

```diff
@@ -0,0 +1 @@
+../../../../dist/bookmarklet.js
```

Comment:
Maybe add a comment here, in the index.html or a note in the README (maybe for the future root-level README) that explains that this is how the `edb` code is getting injected into the app?

Reply:
This is a symlink so we can't add a comment here directly. I do think we should write a root-level README to document some of these things soon. I also don't _think_ this is necessarily the best way to make it available in `ember s`, it's just the simplest thing I could think of. @rwjblue may have better ("more proper") suggestions using the Ember CLI API.